FrontEnd/FrontEnd.Web/Components/Pages/Recipes/SingleRecipe.razor

```diff
@@ -0,0 +1,80 @@
+﻿@page "/recipes/{Id:guid}"
+
+@using FrontEnd.Web.Services
+@using global::Recipes.Public
+@attribute [StreamRendering]
+@inject IRecipeService RecipeService
+
+<style>
+    @@media 
+</style>
+
+@if (_task is not { IsCompleted: true })
+{
+    <PageTitle>Recipe loading</PageTitle>
+    <FluentProgressRing />
+}
```

Comment:
If you're feeling fancy, we could throw in some skeletons while it's loading: https://fluentui-blazor.net/Skeleton

Reply:
I've put some simplified version of the page as a skeleton.